tests/test-bool-2-fail.ql

```diff
@@ -0,0 +1 @@
+bool b = True & 1
```

Comment:
this would fail not because of our semantic check but because w/o an ENDLINE char, parser would fail